core/typedefs.h

```diff
@@ -296,6 +296,10 @@ constexpr T get_num_bits(T x) {
 	return floor_log2(x);
 }
 
+constexpr uint32_t make_fourCC(const char p_sig[5]) {
+	return static_cast<uint32_t>((p_sig[3] << 24U) | (p_sig[2] << 16U) | (p_sig[1] << 8U) | (p_sig[0]));
+}
```

Comment:
Should it be called `MAKE_FOURCC` instead, or perhaps even just `FOURCC()` to match macros like `SNAME()`? That said, it's technically not a macro, so maybe not.

Reply:
I wouldn't capitalize it, we can go with `make_fourcc`.